.test-infra/jenkins/job_beam_Inventory.groovy

```diff
@@ -20,7 +20,7 @@ import common_job_properties
 
 // These jobs list details about each beam runner, to clarify what software
 // is on each machine.
-def nums = 1..8
+def nums = 9..24
```

Comment:
Why `9..24` and not `1..24`?

Reply:
1..8 were in the process of being shut down and removed